mysql/db-init/init.sql

```diff
@@ -1,27 +1,22 @@
-CREATE TABLE translations (
-`id` BIGINT(20) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-`from` VARCHAR(20),
-`source` TEXT,
-`translation` TEXT,
-`created_at` TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)
-);
-
 CREATE TABLE emails2send (
 `id` BIGINT(20) AUTO_INCREMENT PRIMARY KEY,
-`name` TINYTEXT,
-`phone` TEXT,
-`email` TEXT,
-`content` TEXT,
+`uid` VARCHAR(8) NOT NULL,
```

Comment:
why do we need that column?

Reply:
`introduce unique ids to identify library and emails2send entries not to rely on db specific logic`
What do you suggesting instead ? ;)